.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -30,45 +30,43 @@ assignees: ''
         Note that you can point to your own fork here, so you don't actually do changes in the other projects
         before making sure this step works as expected.
 
-- [ ] - 4. Update Kata Containers to use the latest commit from image-rs
+- [ ] - 4. Update Kata Containers to use the latest commit from image-rs, attestation-agent and td-shim
 
-        * https://github.com/kata-containers/kata-containers/blob/CCv0/src/agent/Cargo.toml
+        * image-rs
+          * https://github.com/kata-containers/kata-containers/blob/CCv0/src/agent/Cargo.toml
           * Change the revision
           * Run `cargo update -p image-rs`
-        Note that you can point to your own fork here, so you don't actually do changes in the other projects
-        before making sure this step works as expected.
-
-- [ ] - 5. Update Kata Containers to use the latest attestation-agent
-
-        * https://github.com/kata-containers/kata-containers/blob/CCv0/versions.yaml
-          * Change the version
+          Note that you can point to your own fork here, so you don't actually do changes in the other projects
+          before making sure this step works as expected.
+        * attestation-agent and td-shim
+          * https://github.com/kata-containers/kata-containers/blob/CCv0/versions.yaml
+            * Change the version
 
-- [ ] - 6. Update Kata Containers to use the latest td-shim
+- [ ] - 5. Wait for kata-runtime-payload-ci to be successfully built
+        * After the previous PR is merged wait for the kata-runtime-payload-ci (https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload-after-push.yaml) has completed, so the latest kata-runtime-payload-ci containers the changes
```

Comment:
s/containers/contains ? Yeah, it is containers everywhere all the time :)

Reply:
Thanks for spotting this - at least I didn't autocomplete to containerd that I have a habit of doing 😿